packages/platform/atoms/availability/AvailabilitySettings.tsx

```diff
@@ -389,6 +396,14 @@ export function AvailabilitySettings({
                       copyTime: translations?.copy_times_to ?? t("copy_times_to"),
                       deleteTime: translations?.delete ?? t("delete"),
                     }}
+                    className={{
+                      schedule: customClassNames?.scheduleClassName?.scheduleContainerClassName,
+                      scheduleDay: customClassNames?.scheduleClassName?.scheduleDayClassName,
+                      dayRanges: customClassNames?.scheduleClassName?.dayRangesClassName,
+                      labelAndSwitchContainer:
+                        customClassNames?.scheduleClassName?.labelAndSwitchContainerClassName,
+                      timeRanges: customClassNames?.scheduleClassName?.timeRangesClassName,
+                    }}
```

Comment:
I think we can use the spread operator here

```
className={customClassNames?.scheduleClassName ? {...customClassNames?.scheduleClassName} : {} }
```

Reply:
Will have to rename some inner props inside scheduleClassNames prop but yeah no issues otherwise